extensions/arc/src/extension.ts

```diff
@@ -0,0 +1,34 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as vscode from 'vscode';
+import * as loc from './localizedConstants';
+import { IconPathHelper } from './constants';
+import { BasicAuth } from './controller/auth';
+import { PostgresDashboard } from './ui/dashboards/postgres/postgresDashboard';
+import { ControllerModel } from './models/controllerModel';
+import { PostgresModel } from './models/postgresModel';
+
+export async function activate(context: vscode.ExtensionContext): Promise<void> {
+	IconPathHelper.setExtensionContext(context);
+
+	vscode.commands.registerCommand('arc.managePostgres', async () => {
+		// Controller information
+		const controllerUrl = 'https://0.0.0.0:30080';
```

Comment:
Don't merge these defaults in - just leave them undefined for now in master (even if that mean it's broken there since we don't have a way to launch this command anyways)